src/main/java/HW_2/task2.java

```diff
@@ -0,0 +1,12 @@
+package HW_2;
+
+public class task2 {
+    public static void main(String[] args) {
+        int x = 3;
+        int y = 4;
+
+        System.out.println(x + y);
+        System.out.print(x);
+        System.out.println(y);
+    }
+}
```

Comment:
Implementatiuon is incorrect 
![image](https://github.com/AntBelyi/ALevel_automation/assets/66726924/73cc05eb-dc5e-4785-b13c-711848c148c4)


Yo need only one System.out.println that displays result in one line

Reply:
fixed